README.md

```
@@ -0,0 +1,66 @@
+<br>
+
+## 👀 About 콕콕
+
+혼자서는 흐지부지 끝나버리는 운동...  
+콕콕은 친구들의 도움을 받아 꾸준히 운동하고 싶다는 바람에서 시작됐습니다.  
+가벼운 시도가 꾸준함으로 이어지도록, 사진 인증과 콕 찌르기로 서로를 응원해보세요.  
+자극과 응원이 쌓여 습관이 되는 과정을 함께 만들어가요!  
+<br>
+
+### 📱App Preview
+<img src="./assets/images/screenshot1.png" width="200"/>  <img src="./assets/images/screenshot2.png" style="width:200px; height:432px;"/>  <img src="./assets/images/screenshot3.png" width="200"/>  <img src="./assets/images/screenshot4.png" width="200"/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)